support-frontend/assets/helpers/subscriptionsForms/formReducer.js

```diff
@@ -65,7 +65,7 @@ function createFormReducer(
 
   return (originalState: FormState = initialState, action: Action): FormState => {
 
-    const state = { ...originalState, debugInfo: `${originalState.debugInfo}${JSON.stringify(action)}\n` };
+    const state = { ...originalState, debugInfo: `${originalState.debugInfo} ${JSON.stringify(action)}\n` };
```

Comment:
this extra space makes it a continuation 
https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/AgentReference.html#agent-configuration-file:~:text=so%20any%20line%20that%20begins%20with,and%20starts%20a%20new%20log%20message.